dockstore-cli-integration-testing/src/test/java/io/dockstore/client/cli/GeneralWorkflowIT.java

```diff
@@ -289,10 +289,10 @@ public void testUpdateWorkflowVersion() {
                 "testname", "--workflow-path", "/Dockstore.wdl", "--descriptor-type", "wdl", "--script" });
         Client.main(new String[] { "--config", ResourceHelpers.resourceFilePath("config_file2.txt"), "workflow", "version_tag", "--entry",
             SourceControl.GITHUB.toString() + "/DockstoreTestUser2/hello-dockstore-workflow/testname", "--name", "master",
-            "--workflow-path", "/Dockstore2.wdl", "--hidden", "true", "--script" });
+            "--workflow-path", "/Dockstore2.wdl", "--script" });
```

Comment:
Why is this being removed?

Reply:
Some tests were failing due to setting the default version of an entry to hidden. This is why [this](https://github.com/dockstore/dockstore-cli/pull/46/files#diff-6f92ba8b67e3825333d92f85481441caR106) was changed as well. 

Looks to have been implemented in dockstore/dockstore#3708